### DISCUSSION TITLE
Link-in-bio pages

### DISCUSSION THREAD

**Discussion by steven-tey** [dubinc/dub#319] [Ideas]
Building off of #293 and #376 – planning to add link in bio pages for Dub (e.g. `dub.page/steven`) in the future.

Would this be something y'all would be interested in? 👀

**Comment by tobsowo** [comment]
I'll love to see this.

**Comment by fmerian** [comment]
I personally wouldn't. there are *great* link-in-bio alternatives (think [Bento](https://bento.me/)). above all, I *love* the simplicity and focus of Dub.

**Reply by tobsowo** [reply]
Is Benton open source?

**Reply by fmerian** [reply]
> Is Benton open source?

nope. good objection! then should it be another project (for the sake of simplicity)?

**Reply by fmerian** [reply]
@tobsowo just stumbled upon [openbio](https://github.com/vanxh/openbio) by @vanxh!

**Reply by tobsowo** [reply]
Thank you @fmerian.

**Reply by vanxh** [reply]
> @tobsowo just stumbled upon [openbio](https://github.com/vanxh/openbio) by @vanxh!

thank you. feel free to give any suggestions!
we are gonna build a great open source link in bio page

**Reply by tobsowo** [reply]
@vanxh I will do when I give it a try but can you include screenshot on the website and GitHub page.

**Reply by vanxh** [reply]
> @vanxh I will do when I give it a try but can you include screenshot on the website and GitHub page.

just did!

**Reply by tobsowo** [reply]
> > @vanxh I will do when I give it a try but can you include screenshot on the website and GitHub page.
> 
> just did!

Thank you.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/steven-tey/dub/discussions/319#discussioncomment-6800075

@fmerian, https://bento.me/ has been acquired by Linktree, so I can't say that I'm confident that they'll be around for toooooooo much longer.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/steven-tey/dub/discussions/319#discussioncomment-7093635

If assistive, I have finally located a successor: https://github.com/LinkStackOrg/LinkStack.git.

**Comment by amblerkr** [comment]
The services you're building are always interesting, and I'm looking forward to the profile service you're building, but this is a short address service. It should be later on the roadmap. I think you still have a lot of things to prioritize.
Absorb the good features of competing shortened URL services.

**Comment by abass** [comment]
This would be great! So my wife's goal is to make enough from affiliate revenue via her TikTok account to pay for a LinkTree account, which sounds silly to me because she's using Dub for all her link tracking, so essentially just linking to Dub links on her LinkTree, and she's considering paying LinkTree to get more Dub links to be shown (right now limited at 5).
Feel like a basic version could be a relatively low lift, and could be improved upon to be more competitive in the years to come. But definitely not requesting you half-ass it. Let's just keep links in the Dub fam 😉

**Comment by roke-julian-lockhart** [comment]
I believe that this should be converted into an issue so that we may track implementation of this feature due to the generally positive consensus.

**Comment by kodermax** [comment]
Hi @steven-tey! Do you have any specific deadlines for when you plan to start working on this feature?

**Comment by SirTenzin** [comment]
Coming from #293, I was wondering if this would include a tree of links for the following use cases:

"I am an affiliate who wants one clean link for multiple geographic locations and want to let the user pick": Amazon UK, Amazon US,Bestbuy CA, Bestbuy US

"I am posting job listings for one firm and want a "one-for-all departments" link": SWE, Business Analyst, Accounting Executive etc..

**Comment by imranrkhan13** [comment]
https://linkinpage.techiesaie.com/
check this out
